core/src/script/CGXP/plugins/WFSPermalink.js

```diff
@@ -227,6 +236,10 @@ cgxp.plugins.WFSPermalink = Ext.extend(gxp.plugins.Tool, {
 
         var layerName = state.layer;
         delete state.layer;
+        if (state.show) {
+            this.showFeatures = false;
```

Comment:
Because I am not aware of this syntax... my poor js skills... ;-)

The use-case is as follows:
* If `wfs_show=false` in the request, then `state.show` is defined (and probably equals `false`)
* if `state.show === false`, then `this.showFeatures` has also to be equal to `false`
* By default, `this.showFeatures` is equal to `true`

Regarding the use-case, I don't really get your syntax... (and I have to delete `state.show`...)

To be really strict, I eventually should add

```javascript
if (state.show) {
    if (state.show === false) {
        this.showFeatures = state.show;
    }
    delete state.show;
}
```